src/Repository.php

```diff
@@ -90,7 +90,7 @@ private function checkToArray(Check $check): array
             'group' => $check->group ?? 'other',
             'description' => $check->description,
             'result' => $check->run()->toArray(),
-            'runtimeInMilliseconds' => $start->diffInMilliseconds(),
+            'runtimeInMilliseconds' => (int) $start->diffInMilliseconds(),
```

Comment:
```suggestion
            'runtimeInMilliseconds' => $start->diffInMilliseconds(),
```
Does `diffInMilliseconds` not return an integer?

Reply:
> #### diffIn*
> Yes, the most impactful change is in diffIn* methods. They were returning positive integer by default, they will now return float.

https://carbon.nesbot.com/docs/#api-carbon-3